disnake/iterators.py

```diff
@@ -688,9 +689,11 @@ def create_guild(self, data):
     async def fill_guilds(self):
         if self._get_retrieve():
             data = await self._retrieve_guilds(self.retrieve)
-            if self.limit is None or len(data) < 100:
+            if len(data) < 100:
```

Comment:
shouldn't this be 200, as the endpoint now provides up to 200 guilds?

https://discord.com/developers/docs/resources/user#get-current-user-guilds

Reply:
aefe260be307465e0a90116ec422417d5dcfdbed